main/src/cgeo/geocaching/command/RemoveNotUniqueHelper.java

```diff
@@ -0,0 +1,23 @@
+package cgeo.geocaching.command;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import cgeo.geocaching.models.Geocache;
```

Comment:
Please sort the imports as stated in https://github.com/cgeo/cgeo/wiki/Coding-Conventions